lightning/src/chain/mod.rs

```diff
@@ -134,7 +134,10 @@ pub trait Filter: Send + Sync {
 
 	/// Registers interest in spends of a transaction output identified by `outpoint` having
 	/// `script_pubkey` as the spending condition.
-	fn register_output(&self, outpoint: &OutPoint, script_pubkey: &Script);
+	///
+	/// Optionally, returns any transaction dependent on the output. This is useful for Electrum
+	/// clients to facilitate registering in-block descendants.
```

Comment:
Did you decide against including a block hash to limit returned results to just the current block? Presumably there will be more docs here with an example or such.